spec.html

```diff
@@ -21777,6 +21777,14 @@
         </emu-note>
       </emu-clause>
 
+      <emu-clause id="sec-hostensurecancompilestrings" aoid="HostEnsureCanCompileStrings">
+        <h1>HostEnsureCanCompileStrings( _callerRealm_, _calleeRealm_ )</h1>
+
+        <p>HostEnsureCanCompileStrings is an implementation-defined abstract operation that allows host environments to block certain ECMAScript functions which allow developers to compile strings into ECMAScript code.</p>
+
+        <p>An implementation of HostEnsureCanCompileStrings may complete normally or abruptly. Any abrupt completions will be propagated to its callers. The default implementation of HostEnsureCanCompileStrings is to do nothing.</p>
```

Comment:
> The default implementation of HostEnsureCanCompileStrings is to do nothing.

The default implementation of HostEnsureCanCompileStrings is to unconditionally return an empty normal completion.